src/cli.rs

```diff
@@ -125,6 +126,13 @@ rustup like so: `rustup run <toolchain> <COMMAND...>`. You'll only need to provi
                     how we determine whether a Rust toolchain version is compatible for your crate or not.")
                     .takes_value(false)
                 )
+                .arg(Arg::with_name(id::ARG_RELEASE_SOURCE)
+                    .long("release-source")
+                    .help("Select the rust-releases source to use as the release index")
+                    .takes_value(true)
+                    .possible_values(&["rust-changelog", "rust-dist"])
+                    .default_value("rust-changelog")
+                )
```

Comment:
We should also add `.number_of_values(1)`, otherwise the option may be silently ignored when no value is present. For example: `cargo msrv --min 2018 --bisect --output-format json --release-source`, will fall-back to the default, which may be confusing.

https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.number_of_values

Reply:
Interesting, thanks for mentioning this! I've worked with clap and default values before, but I hadn't considered that use case and that's a good point

Reply:
Hmm, we can skip this for now. I can fix this in one go. Some local testing showed that (at least on my current PC), the method turn `--release-source` without values into an error.